xds/internal/balancer/gracefulswitch/gracefulswitch_test.go

```diff
@@ -1027,16 +874,16 @@ const verifyBalName = "verifyNoSubConnUpdateAfterCloseBalancer"
 const buildCallbackBalName = "callbackInBuildBalancer"
 
 func init() {
-	balancer.Register(bb1{})
-	balancer.Register(bb2{})
-	balancer.Register(vbb{})
-	balancer.Register(bcb{})
+	balancer.Register(mockBalancerBuilder1{})
+	balancer.Register(mockBalancerBuilder2{})
+	balancer.Register(verifyBalancerBuilder{})
+	balancer.Register(buildCallbackBalancerBuilder{})
```

Comment:
Are these still needed if you only access via symbol?

Reply:
Deleted :D.